### PR TITLE
Fix eslint errors

### PR DESCRIPTION
**Pull request by kravets-levko**
Mostly just removed extra empty lines at EoF.

@ranbena and @gabrieldutra There are also some fixes to components you were recently working on, so please check it. Thanks!

### CODE REVIEW

File: client/app/components/dashboards/AddTextboxDialog.jsx [hunk 1]
```diff
@@ -28,8 +28,9 @@ class AddTextboxDialog extends React.Component {
     };
 
     const updatePreview = debounce(() => {
+      const text = this.state.text;
       this.setState({
-        preview: markdown.toHTML(this.state.text),
+        preview: markdown.toHTML(text),
```
Comment (ranbena) [inline client/app/components/dashboards/AddTextboxDialog.jsx]: What lint error is this?

Comment (kravets-levko) [reply client/app/components/dashboards/AddTextboxDialog.jsx]: `react/no-access-state-in-setstate`

File: client/.eslintrc.js [hunk 2]
```diff
@@ -52,6 +54,6 @@ module.exports = {
       ignoreTemplateLiterals: true,
     }],
     "no-else-return": ["error", {"allowElseIf": true}],
-    "object-curly-newline": ["error", {"consistent": true}]
+    "object-curly-newline": ["error", {"consistent": true}],
```
Comment (ranbena) [inline client/.eslintrc.js]: While you're at it - let's turn this annoying thing off?
`"object-curly-newline": "off",`

Comment (ranbena) [reply client/.eslintrc.js]: Actually nevermind - this is my preferred setting but for some reason didn't work in my env.

File: client/app/components/transclude-replace.js [hunk 3]
```diff
@@ -1,26 +0,0 @@
...
-        } else {
-          $element.remove();
-        }
-      });
-    },
-  }));
-}
-
-init.init = true;
-
```
Comment (ranbena) [inline client/app/components/transclude-replace.js]: This is related?

File: client/app/components/dynamic-form/DynamicForm.jsx [hunk 4]
```diff
@@ -47,13 +47,13 @@ export class DynamicForm extends React.Component {
   }
 
   setActionInProgress = (actionName, inProgress) => {
-    this.setState({
+    this.setState(prevState => ({
```
Comment (gabrieldutra) [inline client/app/components/dynamic-form/DynamicForm.jsx]: Cool, I didn't know `prevState` existed here :sweat_smile:

File: client/app/components/tags-control/TagsControl.jsx [hunk 5]
```diff
@@ -79,14 +79,15 @@ export default class TagsControl extends React.Component {
...
-        {this.state.showModal
-          ? <TagsEditorModal
-            tags={tags}
-            availableTags={this.availableTags}
-            close={this.onTagsChanged}
-            dismiss={this.closeEditModal}
-          />
-          : null
+        {
+          this.state.showModal ? (
```
Comment (gabrieldutra) [inline client/app/components/tags-control/TagsControl.jsx]: I would change `? () : null` syntax to `&& ()`, but I guess this is not in the scope, so no big deal :slightly_smiling_face:

File: client/app/components/dynamic-form/DynamicForm.jsx (hunk 4 above)
Comment (kravets-levko) [reply client/app/components/dynamic-form/DynamicForm.jsx]: Me too. I just read the docs :smile:

File: client/app/components/tags-control/TagsControl.jsx (hunk 5 above)
Comment (kravets-levko) [reply client/app/components/tags-control/TagsControl.jsx]: Thanks, fixed :+1:

File: client/app/components/dynamic-form/DynamicForm.jsx (hunk 4 above)
Comment (kravets-levko) [reply client/app/components/dynamic-form/DynamicForm.jsx]: BTW - it's the same warning as @ranbena mentioned above - `react/no-access-state-in-setstate`

Comment (kravets-levko) [reply client/app/components/dynamic-form/DynamicForm.jsx]: Also, `eslint` is smart enough to detect code like this (the same rule):
```javascript
updateCounter() {
  const counter = this.state.counter + 1;
  this.setState({ counter });
}
```
(I tried to modify code this way, but warning didn't disappear :sweat_smile:)

**Pull request merged**